Content.Shared/Weapons/Ranged/Systems/SharedGunSystem.cs

```diff
@@ -389,6 +390,12 @@ protected void SetCartridgeSpent(EntityUid uid, CartridgeAmmoComponent cartridge
         xform.LocalRotation = Random.NextAngle();
         xform.Coordinates = coordinates;
 
+        // decides direction the casing ejects and only when not cycling
+        if (angle != 0f)
+        {
+            angle += 3.7f;
+            ThrowingSystem.TryThrow(entity, angle.ToVec().Normalized() / 100, 5f);
```

Comment:
wut

Reply:
the angle value adds 212 degrees to the direction someone aims at, making the bullet casing fly in a bit of an angle from the opposite of where they are shooting. 
The /100 is a scuffed way of making the bullets drop to the floor a little bit faster and can be removed if you don't like the look of it. 
The 5f is the force at which the casings get thrown.

Bullet spread if shooting at one direction with the /100:
![image](https://github.com/space-wizards/space-station-14/assets/137322659/08b8dcce-87de-4655-abe7-f2c81c58bcb5)

Bullet spread if shooting at one direction without the /100:
![image](https://github.com/space-wizards/space-station-14/assets/137322659/30279fdb-ffd3-4be7-8cf9-9b8b0507fbb2)

Reply:
I added a comment to the code that explains this.